test/shared/src/main/scala/zio/test/mock/TestRandom.scala

```diff
@@ -44,6 +44,30 @@ final case class TestRandom(ref: Ref[TestRandom.Data]) extends Random.Service[An
 
   def shuffle[A](list: List[A]): UIO[List[A]] = Random.shuffleWith(nextInt, list)
 
+  def feedInt(int: Int): UIO[Unit] =
```

Comment:
Ditto on these, I'd update to `feedInts(ints: Int*)` for all of them, and add clear methods as well.